pkg/controller/util.go

```diff
@@ -491,6 +490,10 @@ func MakeCloneTargetPodSpec(image, verbose, pullPolicy string, pvc *v1.Persisten
 					Name:            CLONER_TARGET_PODNAME,
 					Image:           image,
 					ImagePullPolicy: v1.PullPolicy(pullPolicy),
+					SecurityContext: &v1.SecurityContext{
+						Privileged: &[]bool{true}[0],
```

Comment:
Why does the target pod need to run privileged? Is there a more granular capability this pod can be granted?

BTW, good job deducing the problem!

Reply:
Thanks!
It runs privileged as the source pod does. It was added to the source when I add permission issue cloning on minishift. I used to get this error:
"error processing pvc "target-ns/target-pvc": source pod API create errored: pods "source-pod-" is forbidden: unable to validate against any security context constraint: [spec.containers[0].securityContext.securityContext.runAsUser: Invalid value: 0: must be in the ranges: [1000080000, 1000089999]]"

Reply:
Ack. However, running a container privileged is the "sledge hammer" fix. If possible it is preferable to add just the CAPS needed by the container to read from the socket. Maybe that does require a fully privileged container but I'm asking here in case not.

Reply:
@aglitke ?

Reply:
Actually, I am not sure how this is related to removing a label on the pod?

Reply:
It’s not. It’s something I forgot to do before. I can remove it for this PR and submit a different PR for this.